choreographer/src/main/java/eu/arrowhead/core/choreographer/ChoreographerApplicationInitListener.java

```diff
@@ -155,4 +176,19 @@ private UriComponents createUnregisterSystemUri(final String scheme) {
         final String uriStr = CommonConstants.SERVICEREGISTRY_URI + CommonConstants.OP_SERVICEREGISTRY_UNREGISTER_SYSTEM_URI;
         return Utilities.createURI(scheme, coreSystemRegistrationProperties.getServiceRegistryAddress(), coreSystemRegistrationProperties.getServiceRegistryPort(), uriStr);
     }
+    
+    //-------------------------------------------------------------------------------------------------
+    private void configureNetworkAddressVerifier() {
```

Comment:
it assumes that SR is up and running. You should do something like that we did in registering services into SR (trying multiple times, waiting between tries, etc.)

Reply:
I assume because we extend ApplicationInitListener, where `customInit()` only runs after `checkServiceRegistryConnection()`

Reply:
yeah, you are right about this.